### PR TITLE
optimize: remove useless code

### PR DESCRIPTION
**Pull request by zhangzq7**
[remove useless code](https://github.com/seata/seata/commit/ed75c1eca07f3e2c9ecab2bb000c64b12b7624f0)

### CODE REVIEW

**Review by zhangzq7 — COMMENTED** [review]
viewed

**Review by a364176773 — APPROVED** [review]
LGTM

**Comment by a364176773** [comment]
pr 登记一下 https://github.com/seata/seata/tree/develop/changes
中英两版develop.md中进行登记pr的编号和说明及作者信息

File: server/src/main/java/io/seata/server/Server.java [hunk 1]
```diff
@@ -49,7 +47,6 @@ public class Server {
      */
     public static void start(String[] args) {
         // create logger
```
Comment (wangliang181230) [inline server/src/main/java/io/seata/server/Server.java:49]: 这行注释也删了吧。

**Review by jsbxyyx — APPROVED** [review]
LGTM

File: server/src/main/java/io/seata/server/Server.java (hunk 1 above)
Comment (zhangzq7) [reply server/src/main/java/io/seata/server/Server.java]: 已删

**Review by slievrly — APPROVED** [review]
LGTM

**Pull request merged**